src/bikeracks/templates/scripts.js

```diff
@@ -366,13 +360,14 @@ BikeMap.prototype.createMarker = function(state) {
     else {
         // we didn't find a marker, make a new one
         marker = L.marker([state.latitude, state.longitude], {uniqueId: state.rack_id}); 
-        
+        // Figure out marker's status
```

Comment:
This is not really a useful comment. After you read it, you don't know anything that isn't already obvious from the code.

Maybe `// True implies that this rack is approved` ?

Reply:
Done.